src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -225,7 +225,17 @@ export class NavigationTracker extends BaseTracker {
    */
   isHeaderMenuBarLink(element: HTMLElement): boolean {
     const header = document.getElementsByTagName("header")[0];
```

Comment:
I think because there will only ever be one header or nav element on a page... 

 We can use 
`document.querySelector("header")`
`document.querySelector("header");` as this will always just return one element, rather than returning all elements, then selecting the first one?

Reply:
yep, makes sense. I have updated now